pulsar-transaction/coordinator/src/test/java/org/apache/pulsar/transaction/coordinator/MLTransactionMetadataStoreTest.java

```diff
@@ -55,12 +60,19 @@
 
 public class MLTransactionMetadataStoreTest extends MockedBookKeeperTestCase {
 
+    private ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(3);
```

Comment:
1. It's better to add a single test that adds batch transaction log and then can open a new cursor for the transaction log managedLedger and read the batch log. then the TransactionCoordinator can recover successfully
2. add a test for the log delete